docs/user-guide/configuring.md

```diff
@@ -208,19 +208,19 @@ To specify globals using a comment inside of your JavaScript file, use the follo
 /* global var1, var2 */
 ```
 
-This defines two global variables, `var1` and `var2`. If you want to optionally specify that these global variables should never be written to (only read), then you can set each with a `false` flag:
+This defines two global variables, `var1` and `var2`. If you want to optionally specify that these global variables can be written to (rather than only being read), then you can set each with a `writeable` flag:
```

Comment:
This documentation was previously misleading -- it implied that "writeable" was the default for global comments like `/* global foo */`, but in fact the default seems to be "readable".